src/values/expressions/ObjectExpression.js

```diff
@@ -9,7 +9,8 @@ import getValue from './index';
 export default function extractValueFromObjectExpression(value) {
   return value.properties.reduce((obj, property) => {
     const object = Object.assign({}, obj);
-    if (property.type === 'SpreadProperty') {
+    // Support types: SpreadProperty and ExperimentalSpreadProperty
+    if (/SpreadProperty/.test(property.type)) {
```

Comment:
```suggestion
    if (/^(?:Experimental)?SpreadProperty$/.test(property.type)) {
```
this seems a bit more robust

Reply:
@ljharb I like the way you regex.